series/brno-pyvo/events/2019-03-28-ucici.yaml

```
@@ -0,0 +1,41 @@
+city: brno
+start: 2019-03-28 19:00:00
+name: Brněnské Pyvo
+topic: Učící
+venue: artbar
+description: |
+    Pojďme si popovídat o učení.
+    Ne o tom strojovém, které je teď tak populární, ale o tom obyčejném, lidském.
+    Jak a proč se učit programovat?
+    Ať už od začátků, nebo, když už to člověk umí, jak se vzdělávat dál?
+    Jak a proč učit někoho jiného?
+    Kdy je dobré začít?
+    Těšíme se na vás a na vaše příspěvky do diskuze!
+
+    ------
+
+    Let's talk about learning!
+    Not machine learning which is so popular nowadays. The normal one, of a human.
```

Tohle mi zní divně, dal bych tam třeba:
```suggestion
    Not machine learning which is so popular nowadays, but the normal kind – human learning.
```